sdk/keyvault/azure-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/models/KeyCreateOptions.java

```diff
@@ -127,7 +127,7 @@ public KeyCreateOptions setExpires(OffsetDateTime expires) {
      *
      * @return the expires UTC time.
      */
```

Comment:
There are other methods in this class that doesn't have the `get` prefix like `notBefore`, `keyType`, `tags` .
We don't need to update these?

Reply:
yup, updated.